spec/support/helpers.rb

```diff
@@ -139,6 +139,14 @@ def expect_query_with_consistency(matcher, consistency)
         expect(cequel.client).to have_received(:execute).
           with(matcher, hash_including(:consistency => consistency))
       end
+
+      def expect_query_with_options(matcher, options)
+        allow(cequel.client).to receive(:execute).and_call_original
+        yield
+        expect(cequel.client).to have_received(:execute).
+          with(matcher, hash_including(options))
+      end
+
```

Comment:
Now that I look at it, I could probably clean up the method that checks for consistency to reuse this one as well.